scripts/airgap/image-list.txt

```diff
@@ -1,5 +1,5 @@
-docker.io/rancher/klipper-helm:v0.6.6-build20211022
-docker.io/rancher/klipper-lb:v0.3.4
+docker.io/rancher/klipper-helm:v0.6.8-build20211123
+docker.io/rancher/klipper-lb:v0.3.5
```

Comment:
this also needs to be updated in the servicelb codebase

Reply:
Done.